tools/apidiff/exports/exports.go

```diff
@@ -110,10 +118,15 @@ func (c *Content) addConst(pkg Package, g *ast.GenDecl) {
 				// const FooConst = FooType("value")
 				co.Type = pkg.getText(ce.Fun.Pos(), ce.Fun.End())
 				v = pkg.getText(ce.Args[0].Pos(), ce.Args[0].End())
+			} else if ce, ok := vs.Values[0].(*ast.BinaryExpr); ok {
+				// const FooConst = "value" + Bar
+				co.Type = "*ast.BinaryExpr"
+				v = pkg.getText(ce.X.Pos(), ce.Y.End())
 			} else {
-				panic(fmt.Sprintf("unhandled case for adding constant: %s", pkg.getText(vs.Pos(), vs.End())))
+				panic("unhandled case for adding constant")
```

Comment:
Can we keep the text for the line in the panic message?

Reply:
Done